.travis.yml

```diff
@@ -118,6 +119,8 @@ script:
   - ./config no-threads shared enable-ssl3 enable-ssl3-method -g --prefix=$OPENSSL_PREFIX -DPURIFY > build.log 2>&1 || (cat build.log && exit 1)
   - make -j$JOBS > build.log 2>&1 || (cat build.log && exit 1)
   - sudo make PATH=$PATH install_sw > build.log 2>&1 || (cat build.log && exit 1)
+  - cd ../lua-resty-signal/
+  - make
```

Comment:
This will require changes in our [EC2 test cluster](https://openresty.org/en/ec2-test-cluster.html) as well. Better don't do it.

Reply:
OK